app/scripts/modules/core/src/pipeline/config/templates/v2/PipelineTemplatesV2.less

```diff
@@ -0,0 +1,13 @@
+.pipeline_templates_error_banner {
+  margin: 1em 0;
+
+  i.fa-exclamation-triangle {
+    padding-right: 8px;
+  }
+}
+
+.pipeline_template_actions {
```

Comment:
Just wondering about the convention here, does the project practice underscores instead of dashes? Or is this a similar to BEM with the double underscore to signify an element in a block?

Reply:
Just discovered that we use BEM according to our Conventions wiki page.  I'll revisit the names here.